README.md

```diff
@@ -83,6 +87,12 @@ func main() {
 	if err := defaults.Set(obj); err != nil {
 		panic(err)
 	}
+	// Or with a specific tag name (without setting defaults.TagName globally):
+	/*
+	if err := defaults.SetWithTag(obj, "some_tag_name"); err != nil {
+	    panic(err)
+	}
+	*/
```

Comment:
It's strange to find this commented in an example

Reply:
I could add additional struct tags to the example structs to demonstrate as part of the example?

Reply:
Whoops- if it wasn't apparent, above is me; apologies!

Reply:
FIXED:
* Example struct given additional tag `default_alt` to demonstrate `SetWithTag` usage
* Is now an additional call at the end of `main()` in the example instead of only commented